vendor/revisions.json

```diff
@@ -1,14 +1,14 @@
 {
   "v16": [
-    "16.3",
-    "5ea106b2583285849784e774b39d62eb2615bd5d"
+    "16.4",
+    "a0206fd86672957d6dc714ae957184feb139f5db"
```

Comment:
Were there any non-trivial conflicts in the Postgres PRs that you had to resolve?

It would be also helpful to add links to Postgres PRs in this PR description

Reply:
Nothing non-trivial. We got the upstream commit from the MVCC bug that @hlinnaka fixed and put in our fork before it was in master, and there was a comment conflict.

Reply:
I still see some conflicts and `check-codestyle-rust` non-passing for some reason. Otherwise, LGTM, so I'm ready to approve as soon as it's all-green